tests/cpp/test_gpu_view.cpp

```diff
@@ -1447,61 +1447,77 @@ TEST_F(GpuViewTest, FusionSumViewSchedule) {
 
 // Make sure matching reshapes are segmented into the same kernel
 TEST_F(GpuViewTest, FusionReshapeMagicSchedule1) {
-  auto fusion_ptr = std::make_unique<Fusion>();
-  Fusion& fusion = *fusion_ptr.get();
-  FusionGuard fg(&fusion);
+  auto fusion = std::make_unique<Fusion>();
+  FusionGuard fg(fusion.get());
```

Comment:
Sorry for mixing in some cosmetic changes like this. Happy to revert if they are annoying enough.